test/bodyHelper.js

```diff
@@ -0,0 +1,18 @@
+import TestHelper from './TestHelper';
```

Comment:
You are importing an instance. it should not start in caps. `testHelper `

Reply:
Thanks a lot for the review, but the TestHelper is a class and I thought classes were meant to be Pascal cased.

Reply:
I have refactored this to an object. Thanks